src/components/modules/blockEvents.ts

```diff
@@ -16,11 +16,18 @@ export default class BlockEvents extends Module {
    * @param {KeyboardEvent} event - keydown
    */
   public keydown(event: KeyboardEvent): void {
+    /**
+     * return if isPreventDefaultSupported is true
+     */
+    const tool = this.Editor.BlockManager.currentBlock.class;
+
+    if (this.Editor.Tools.isPreventDefaultSupported(tool)) {
```

Comment:
it's better to access it via the  `INTERNAL_SETTINGS` map. It will help us to rename such settings in the future